addons/website_slides/models/slide_channel.py

```diff
@@ -147,11 +147,12 @@ def _default_access_token(self):
         help='Applied directly as ACLs. Allow to hide channels and their content for non members.')
     partner_ids = fields.Many2many(
         'res.partner', 'slide_channel_partner', 'channel_id', 'partner_id',
-        string='Members', help="All members of the channel.", context={'active_test': False}, depends=['channel_partner_ids'])
+        string='Members', help="All members of the channel.", context={'active_test': False}, depends=['channel_partner_ids'], copy=False)
     members_count = fields.Integer('Attendees count', compute='_compute_members_count')
     members_done_count = fields.Integer('Attendees Done Count', compute='_compute_members_done_count')
     is_member = fields.Boolean(string='Is Member', compute='_compute_is_member')
-    channel_partner_ids = fields.One2many('slide.channel.partner', 'channel_id', string='Members Information', groups='website.group_website_publisher', depends=['partner_ids'])
+    channel_partner_ids = fields.One2many('slide.channel.partner', 'channel_id', string='Members Information', groups='website.group_website_publisher', depends=['partner_ids'],
```

Comment:
One2Many fields are not copied by default ;).

Reply:
It has stopped Adding duplicated members
Tested on this Build http://644311-38235-f72471.runbot27.odoo.com/web#action=894&model=slide.channel&view_type=kanban&cids=&menu_id=629
![Screenshot (26)](https://user-images.githubusercontent.com/16624719/66422423-4d1ed080-ea27-11e9-9674-aa79d27e0a90.png)


I will check again if needed.

Reply:
Yes, but the copy=False is only needed on the partner_ids field and not on the channel_partner_ids one I think.

Reply:
Checking this (Once done with https://github.com/odoo/odoo/pull/38239)

Reply:
It seems you are right (also I missed `depends=['partner_ids']` there)
Thanks :+1:

Reply:
@Feyensv Looking at copy_data in models.py it seems that o2m and m2m are copied.

Reply:
@tde-banana-odoo o2m can be copied, but by default, they are copy=False
https://github.com/odoo/odoo/blob/4ad7a99df0a25122cf6872f880e07b6148c82ea8/odoo/fields.py#L2781-L2787

Reply:
Indeed. However I prefer being explicit and I prefer to have copy=False on all partner-related fields.